blog-service/2023-04-07-search.md

```diff
@@ -15,7 +15,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 New - We're excited to announce automatic log level detection for Log Search queries, helping you to quickly identify anomalies without having to search through large volume of logs to find high severity issues.
 
-With this update, you can visualize and filter log-level distribution in the Histogram results and Messages table messages of specific log level(s) in the same view. [Learn more](/docs/beta/search-log-level).
+With this update, you can visualize and filter log-level distribution in the Histogram results and Messages table messages of specific log level(s) in the same view. [Learn more](/docs/search/get-started-with-search/search-page/log-level).
```

Comment:
I think breaking this sentence into two parts could make it easier to read and understand. For example, you could say: 
With this update, you can now visualize and filter log-level distribution in both the **Histogram** results and **Messages** table. This allows you to view messages of specific log level(s) in the same view.

Reply:
will do